portfolio/src/main/webapp/index.html

```diff
@@ -10,6 +10,11 @@
 </head>
 
   <body>
+    <ul>
+        <li class="fonts"><a href="https://mwong-sps-summer22.appspot.com/">Matthew Wong</a></li>
+        <li class="fonts" style="float:right"><a class="active" href="learnings.html">Learnings</a></li>
+    </ul>
```

Comment:
Optional: Maybe add a border + background color, or something to indicate this text is actually a link to the home page.

Reply:
Added hover that changes it to blue, looks nicer now 😎